lib/msal-node/test/test_kit/StringConstants.ts

```diff
@@ -348,6 +350,22 @@ export const AUTHENTICATION_RESULT_DEFAULT_SCOPES = {
     },
 };
 
+export const MANAGED_IDENTITY_RESOURCE =
+    "URI_of_the_managed_identity_assigned_to_the_azure_resource";
+
+export const MANAGED_IDENTITY_CLIENT_ID =
+    "unique_identifier_generated_by_Azure_AD_for_the_azure_resource";
+
+export const DEFAULT_MANAGED_IDENTITY_AUTHENTICATION_RESULT: ManagedIdentityResult =
+    {
+        accessToken: TEST_TOKENS.ACCESS_TOKEN,
+        expiresOn: TEST_TOKEN_LIFETIMES.DEFAULT_EXPIRES_IN,
```

Comment:
This must be a bug. `expiresOn` is a date but expires_in is a number (of seconds)

Reply:
I believe expiresOn should be a number. It's returned from the managed identity as a number. See [here](https://learn.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=portal%2Chttp). We also return it as a number in lib/msal-node/src/utils/ManagedIdentityUtils.ts

Reply:
Then rename the variable as `expiresIn`?

`expiresOn` - name suggests this is a date
`expiresIn` - name suggestions this is a duration in seconds

When it stores tokens in the cache, MSAL uses dates.